### DISCUSSION TITLE
Future action for specific image

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/publishpress-future#628] [Feature Requests]
They offer a plugin that can deactivate an article on a certain date. Unpublish WordPress Posts

I need a solution that allows you to set the date when either the whole article (or just the photos of an article) will be taken offline right at the beginning when you create a post. The article without photos would then remain online.

Not all photos in the articles can be used indefinitely. (Copyright)

So if I create an article on 6 December 2023, for example, it should be possible to specify that the article (the text) remains online, but all photos (or only certain photos) of an article are removed on 6 March 2024.

It would be best to have a date picker for each photo. If nothing is selected, the photo remains online indefinitely. If you select a date for the photo when you post it, this is the future deletion date.

https://secure.helpscout.net/conversation/2455117534/217348?folderId=6932143

**Comment by andergmartins** [comment]
Maybe a duplication of #541
